### ISSUE TITLE
how to delete a tweet by id?

### ISSUE THREAD

**Issue by devgaucho** [noweh/twitter-api-v2-php#40]
the endpoint is https://api.twitter.com/2/tweets/:id

documentation [here](https://developer.twitter.com/en/docs/twitter-api/tweets/manage-tweets/api-reference/delete-tweets-id)

what is the lib method?

**Comment by devgaucho** [comment]
UPDATE:
the correct code is the following:

```php
$client->tweet()->delete($id)->performRequest()
``

**Comment by noweh** [comment]
Indeed, the api has been redesigned since February 2022, sorry for the inconvenience
